### DISCUSSION TITLE
2.8.7 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#4814] [讨论]
· 使用新的正版登录方式，以提高安全性（#122，@zkitefly，PR @Pigeon0v0）
· 略微减轻了左下角提示遮挡 UI 的情况（#4082，@qndm）
· 需要更新密钥的同时显示识别码（#4188，@Louis-Harsune）
· Workflow 编译支持替换部分 Secret（#4228，PR @allMagicNB）
· 在老版本的反馈失败提示中增加了更新入口（#4662，@WForst-Breeze）
· 新主页预设：PCL 主页市场（#4668，由 @JingHai-Lingyun 制作）
· 支持从官方源下载 Authlib-Injector（#4703，@bmmNbhjd）
· 支持加载 WebP 格式的 Mod 图标（#4734，@wuliaodexiaoluo，PR @tangge233）
· 修改版本隔离设置时会显示警告（#4778，@shenbimicro233）
· 为愚人节版本添加发布年份显示（#4797，@Crotina）
· 不再完全支持 32 位系统，若在 32 位系统下运行会显示环境警告
· 请求 Mod 下载站 API 失败时会进行自动重试
· 修复了以下 23 个 Bug
　　若 Mod 缓存有误，则再也无法加载 Mod 信息（#3433，@SDHsuifeng）
　　在非 NTFS 磁盘上无法为远古版本下载资源（#3684，@misividkoukou）
　　下载游戏可能失败：版本 json 不规范（#3710，@Afdian-User-1145）
　　[关闭文件校验] 选项可能失效（#3792，@wuliaodexiaoluo）
　　游戏内打开文件夹的路径可能有误（#3868，@EndlessEmptiness，PR @Pigeon0v0）
　　[检查更新] 按钮不会积极地获取更新（#4217，@allMagicNB）
　　access_token 过期后启动游戏不会自动刷新（#4333，@allMagicNB）
　　Mod 标题太长会点不到升级图标（#4465，@AlbertInf299）
　　local:MyIconTextButton 的 ColorType 属性无效（#4538，@MFn233）
　　无法删除远古版本 MC 与文件夹符号链接（#4549，@AptS-1547）
　　远古版本在开启版本隔离后没有声音（#4628，@zec896219）
　　删除 Mod 后报错（#4691，@Zecci9）
　　Mod 筛选器的显示条件有误（#4701，@yizhi9jiyan9）
　　版本设置的自动分配和全局设置的行为不一致（#4752，@hxabcd）
　　双击 Mod 更新按钮会导致更新无法正常结束（#4754，@addion7）
　　有多个下载任务时结束任务会出现意料之外的行为（#4766，@MacroMeng）
　　检查的可更新的 Mod 存在遗漏（#4805，@Lmy1503）
　　显示前置 Mod 时可能出错（#4806，@Molacs）
　　代码中的 Unavailable 拼写错误
　　在开启 [关闭文件校验] 时手动补全文件，会忽略设置并正常进行文件校验
　　请求速率会超出 BMCLAPI QPS 限制
　　每次打开均会尝试联网获取本地 Mod 信息
　　部分 Mod 的图标无法加载

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.8.7'>2.8.7</a>.</em>

**Comment by 3gf8jv4dv** [comment]
Are the next milestones still focused on fixing bugs, or include some new features? 🤔

**Reply by tangge233** [reply]
More new features I think.
(There are still some PRs waiting for merge…)

**Comment by LTCatt** [comment]
下一个版本的计划：

- 推送正式版（如果这个版本有大 bug 会补一个临时更新）
- #4707 
- 解决中优先度的 issue（现在堆得还是太多了）
- 也许会做 #4810
